docs/data.rst

```diff
@@ -287,9 +287,10 @@ Sample response with link header
       ]
 
 
-Sort submitted data of a specific form using existing fields
--------------------------------------------------------------
-Provides a sorted list of json submitted data for a specific form by specifing the order in which the query returns matching data. Use the `sort` parameter to filter the list of submissions.The sort parameter has field and value pairs.
+Sort form data against a column(field)
+----------------------------------------
+Returns submitted data for a specific form in a sorted order. Use the `sort` parameter to filter the list of submissions.
+The sort parameter is takes a field(form column) as the key and select value pairs as demonstrated below.
```

Comment:
I think the previous version was clear enough... I don't see the need to change it